template/config/init.sls

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
How about line 1 being a breadcrumb?

```
1  # {{ tpldir }}/config/clean.sls
2 # # -*- coding: utf-8 -*-
```

Reply:
Even better:

```sls
# {{ tplfile }}
# -*- coding: utf-8 -*-
```

Reply:
nice

Reply:
OK, have spoken to @noelmcloughlin directly and we've decided that there's no need for this.